### DISCUSSION TITLE
Cool feature ideas

### DISCUSSION THREAD

**Discussion by karkir0003** [community/community#32365] [Other Feature Feedback, Questions, & Ideas]
Hello Github Community,
I hope you're doing well. I'm a really big fan of GitHub and the positive impact its features have on the development community. I thought of a couple of interesting potential feature ideas that the GitHub Engineering team could potentially develop (time permitting though I guess):

* For coding interviews, since Github has collaborative editor feature, what might be helpful for people who want to say prep for mock interviews could be for GitHub to have a Microsoft Teams integration where the user can be like "Conduct Mock interview", select a friend who has github account (or enter email account) and a collaborative editor can be automatically created. A user can have the ability to integrate a repo and once the mock interview concludes, the recording (with user consent) can be uploaded to the repo automatically. This feature could potentially help increase GitHub's accessibility to the community

* Drag and Drop UI to create CI/CD flows and autogenerate the yml for custom Github Action workflow (maybe for long term goal though)

I was wondering if there was anyne from the Github team I could talk to about this.

**Comment by farisdurrani** [comment]
Cool idea! Maybe the drag and drop can also recommend common CI/CD commands as you fill it
